radio/src/gui/colorlcd/model_setup.cpp

```diff
@@ -676,6 +676,16 @@ class ModuleWindow : public FormGroup {
           }
           else {
             new NumberEdit(this, grid.getFieldSlot(2,0), -128, 127, GET_SET_DEFAULT(g_model.moduleData[moduleIdx].multi.optionValue));
+
+            //Show RSSI next to RF Freq Fine Tune
+            if (getMultiOptionTitle(moduleIdx) == STR_MULTI_RFTUNE)
+            {
+              new DynamicText(this, grid.getFieldSlot(2, 1), [=] {
+                  char msg[64] = "";
```

Comment:
Is there any significance to this being 64? I noticed all the other instances are the same size... I was going to do this 16 as that was more than sufficient... then thought otherwise.